libs/packages/layouts/src/lib/feature/subheader/subheader.module.ts

```diff
@@ -34,13 +35,15 @@ import { IconModule } from '@gsa-sam/ngx-uswds-icons';
     SdsSubheaderActionsComponent,
     SdsSubheaderDrawerComponent,
     SdsDrawerContentComponent,
+    SdsSubheaderWrapperComponent
 
   ],
   declarations: [
     SdsSubheaderComponent,
     SdsSubheaderActionsComponent,
     SdsSubheaderDrawerComponent,
-    SdsDrawerContentComponent
+    SdsDrawerContentComponent,
+    SdsSubheaderWrapperComponent
```

Comment:
This change seems to have been missed - could you also add chevronLeft to the bootstrap icon imports in line 30/31? I see chevronLeft icon being used in line 4 of `sds-subheader-wrapper.component.html` in the git diff. This issue likely won't come up in the demo because we have every single icon imported in `documentaion.module.ts` file, but will be apparent if users attempt to use this module without importing chevronLeft.

Reply:
Not sure I understand. Are you saying you can see chevron left working as intended without importing it? That would mean that the icons aren't being lazy loaded. Just to confirm, you also removed the icons import in`documentation.module.ts`?

Reply:
Missed the load all. Just added the chevron into the subheader module itself. Thank you for pointing that out.